bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -7,13 +7,28 @@
 	<description>The Netatmo binding integrates Weather Station with companion modules, Healthy Home Coach, Thermostat Plug
 		and Welcome Camera.</description>
 
+
 	<config-description>
-		<parameter name="backgroundDiscovery" type="boolean" required="false">
-			<label>Background Discovery</label>
-			<description>If set to true, the device and its associated modules are updated in the discovery inbox at each API
-				call run to refresh device data. Default is false.</description>
+		<parameter name="features" type="text" required="true" multiple="true">
+			<label>Feature Area</label>
+			<description>Defines the set of capabilities you want to operate on.</description>
+			<options>
+				<option value="AIR_CARE">Air Care</option>
+				<option value="WEATHER">Weather</option>
+				<option value="ENERGY">Energy</option>
+				<option value="SECURITY">Security</option>
+			</options>
+			<limitToOptions>true</limitToOptions>
+			<default>WEATHER</default>
+		</parameter>
+
+		<parameter name="readFriends" type="boolean">
+			<label>Access Guests</label>
+			<description>For Weather Stations : A friend gave you access to their Netatmo Weather Station.</description>
```

Comment:
Sorry, late finding after looking at newly generated i18n file (remember to update also):
```suggestion
			<description>For Weather Stations: A friend gave you access to their Netatmo Weather Station.</description>
```